fix: make upgrade (#255)
sse-secure-systems/connaisseur: docs/basics.md

> pod/hello-world created
```

### Upgrade

> The configuration of a running Connaisseur instance can be  updated by a Helm upgrade of the current release:

Reviewer: ```suggestion
The configuration of a running Connaisseur instance can be updated by a Helm upgrade of the current release:
```

> Should you want to make some adjustments to the Connaisseur configuration while Connaisseur is already running, Helm can be used to upgrade the current release.

Reviewer 2: ```suggestion
The configuration of a running Connaisseur instance can be  updated by a Helm upgrade of the current release:
```
the proposed sentence is a bit  backwards and unnecessarily lengthy
